pipeline/handler.py

```diff
@@ -0,0 +1,28 @@
+"""Lambda Handler"""
+
+import traceback
+from os import environ as ENV
+from dotenv import load_dotenv
```

Comment:
you're not using these so you don't need to import them

Reply:
Good catch